src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -56,9 +56,7 @@ public function __construct(FormTypeInterface $innerType, array $typeExtensions
     }
 
     /**
-     * Returns the prefix of the template block name for this type.
-     *
-     * @return string The prefix of the template block name
+     * {@inheritdoc}
```

Comment:
I think it should be removed. I don't see the point of having this method in the resolved type as it must equal the inner one. So overwriting it makes no sense either.